fastlane/Helpers/Distribution.swift

```diff
@@ -25,4 +25,27 @@ enum Distribution {
             debug: .userDefined(true)
         )
     }
+
+    static func uploadToAppStore() {
+        let environment: Constant.Environment = .production
```

Comment:
How about this?
```swift
static func uploadToAppStore(environment: Constant.Environment = .production)
```

We can support upload staging build to AppStore as well, some project did this.